include/natalie/integer_object.hpp

```diff
@@ -77,14 +76,14 @@ class IntegerObject : public Object {
     static T convert_to_native_type(Env *env, Value arg) {
         auto integer = arg->to_int(env);
         if (integer->is_bignum())
-            env->raise("RangeError", "bignum too big to convert into `{}'", typeinfo<T>().name());
+            env->raise("RangeError", "bignum too big to convert into `{}'", typeid(T).name());
```

Comment:
I have looked into `typeid` when I wrote this code, and this is not a suitable alternative. Instead of the name, it returns a one character name that may or may not differ between compilers/architectures.

For example: `int32_t` returns "i", `uint32_t` returns "j", `int8_t` return "a", `uint8_t` return "h".
I don't think I would understand what an error message "to big to convert into `j'" would mean.